client/my-sites/sharing/connections/services-group.jsx

```diff
@@ -19,36 +19,45 @@ import ServicePlaceholder from './service-placeholder';
  */
 const NUMBER_OF_PLACEHOLDERS = 4;
 
-const SharingServicesGroup = ( { connections, services, title } ) => (
-	<div className="sharing-services-group">
-		<SectionHeader label={ title } />
-		<ul className="sharing-services-group__services">
-			{ services.length
-				? services.map( ( service ) => (
-					<Service key={ service.ID } connections={ connections } service={ service } />
-				) )
-				: times( NUMBER_OF_PLACEHOLDERS, ( index ) => (
-					<ServicePlaceholder key={ 'service-placeholder-' + index } />
-				) )
-			}
-		</ul>
-	</div>
-);
+const SharingServicesGroup = ( { connections, isFetching, services, title } ) => {
+	if ( services.length || isFetching ) {
```

Comment:
Could avoid indentation of most of the function by doing the early return of `null` instead:

```js
if ( ! services.length && ! isFetching ) {
	return null;
}
```